### PR TITLE
Feature/search api new auth

### PR DESCRIPTION
**Pull request by gedge**
### What

updated search api tests to handle new auth

### How to review

check and run the tests against the new search-api (separate PR)

### Who can review

!gedge

### CODE REVIEW

File: searchAPI/delete_search_Index_test.go [hunk 1]
```diff
@@ -6,33 +6,36 @@ import (
 	"testing"
 
 	"github.com/ONSdigital/dp-api-tests/testDataSetup/elasticsearch"
+	"github.com/ONSdigital/go-ns/common"
 	"github.com/ONSdigital/go-ns/log"
 	"github.com/gavv/httpexpect"
 	. "github.com/smartystreets/goconvey/convey"
 )
 
+const resourceNotFound = "esource not found"
```
Comment (CarlHembrough) [inline searchAPI/delete_search_Index_test.go]: intentional or typo?

File: searchAPI/get_dimension_search_test.go [hunk 2]
```diff
@@ -56,49 +61,51 @@ func TestSuccessfullyGetDimensionViaSearch(t *testing.T) {
...
 						if count, ok := response.Value("count").Raw().(float64); ok {
 							if count != 0 {
-								tryAgain = false
+								foundCount = true
+								break searchCompleteLoop
 							}
 						}
-						time.Sleep(100 * time.Millisecond) // Don't want to batter the api
+						log.DebugC("searchCompleteLoop", "got empty search results", log.Data{"resp": response.Raw()})
+						time.Sleep(retryPause) // Don't want to batter the api
```
Comment (CarlHembrough) [inline searchAPI/get_dimension_search_test.go:98]: This is a winning comment

**Review by CarlHembrough — APPROVED** [review]
LGTM

File: searchAPI/delete_search_Index_test.go (hunk 1 above)
Comment (gedge) [reply searchAPI/delete_search_Index_test.go]: intentional - I'll add a comment:
`// this is to cover "Resource" and "resource" not found`

File: searchAPI/get_dimension_search_test.go (hunk 2 above)
Comment (gedge) [reply searchAPI/get_dimension_search_test.go:98]: all credit to Nathan

**Pull request merged**